pom.xml

```diff
@@ -10,4 +10,16 @@
     <module>org.jacoco.build</module>
   </modules>
 
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.sonarsource.scanner.maven</groupId>
+          <artifactId>sonar-maven-plugin</artifactId>
+          <version>4.0.0.4121</version>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+  </build>
```

Comment:
Should be in `org.jacoco.build/pom.xml` because CI executes `mvn -f org.jacoco.build` which doesn't inherit from this `pom.xml`.